### DISCUSSION TITLE
Angular issue when using pixi.js@6.0.*

### DISCUSSION THREAD

**Discussion by donmahallem** [pixijs/pixijs#7405] [Q&A]
I do know that this might not be an pixi exclusive issue but I can't blame angular currently either directly for this as during pixi setup something gets messed up.

### Expected Behavior
It should render the provided sprite inside the angular component.
### Current Behavior
Doesn't initialize the renderer correctly

### Possible Solution
---
### Steps to Reproduce

Setup a simple pixi application with a sprite and attach it to the component and try to call render.
Neither Application nor autoDetectRenderer do work.
### Environment

- **`pixi.js` version**: 6.0.x_
- **Browser & Version**: Chrome 89
- **OS & Version**: Windows
- **Running Example**: https://stackblitz.com/edit/angular-zvqwsh?file=src/app/hello.component.ts

**Comment by bigtimebuddy** [comment]
Hi @donmahallem, thanks for pointing this out. In general, integration with other DOM frameworks like React or Angular isn't a priority for this project. There are lots of resources like Stack Overflow and other places for tips on how to get this to work. Also, for support-related questions, if you wouldn't mind, please use the Discussion section not Issues. 

That being said, here is the important bit:

```typescript
import { Application } from "@pixi/app";
import { Sprite } from "@pixi/sprite";
```
The easiest solution is to do this:

```typescript
import { Application, Sprite } from "pixi.js";
```

![Screen Shot 2021-04-14 at 9 40 02 AM](https://user-images.githubusercontent.com/864393/114747169-66cc6280-9d05-11eb-8ab1-d840e2108f14.png)

Here's the reason why this works. If you're using the sub-packages (`@pixi/*`) you should consult https://pixijs.io/customize to make sure that Renderer plugins are properly installed. I suspect you're missing these bits:

```typescript
import { Renderer } from '@pixi/core';
import { BatchRenderer } from '@pixi/core';
Renderer.registerPlugin('batch', BatchRenderer);
```

The `batch` plugin is used for rendering Sprites. If that's not installed, you get the behavior you were seeing.
